saleor/graphql/product/types/products.py

```diff
@@ -1590,8 +1590,10 @@ class Category(ModelObjectType):
         lambda: CategoryCountableConnection,
         description="List of ancestors of the category.",
     )
-    products = ConnectionField(
+    products = FilterConnectionField(
         ProductCountableConnection,
+        filter=ProductFilterInput(description="Filtering options for products."),
```

Comment:
IMHO, both parameters should get an additional label `ADDED_IN_39`
https://github.com/saleor/saleor/blob/main/saleor/graphql/core/descriptions.py#L22
(At the end of description)
Example:
```
        description=(
            "The delivery method selected for this order."
            + ADDED_IN_31
       )
```

Reply:
Label "ADDED_IN_39" has been added to descriptions of filter and sortBy parameters.

Reply:
We just release 3.9, can you update it to ADDED_IN_310?